docs/src/modules/javascript/partials/eventsourced.adoc

```diff
@@ -6,8 +6,6 @@ An Event Sourced Entity must not update its in-memory state directly as a result
 
 When you need to read state in your service, ask yourself _what events should I be listening to_?. When you need to write state, ask yourself _what events should I be emitting_?
```

Comment:
Not sure why the todo is being removed, I don't see an illustration added?

Reply:
Sorry - the TODO is appearing in the live PROD docs for some reason. It looked correct to me, so this seemed best for now.
https://docs.akkaserverless.dev/java/eventsourced.html

Reply:
How strange, it does seem to be formatted correctly. Sometimes if you delete it and do it again that fixes it.

Reply:
ok - I add them back in.